R/get_pvol.R

```diff
@@ -200,13 +200,14 @@ read_pvol_from_url_per_param <- function(urls, ..., call = rlang::caller_env())
             httr2::req_options(ssl_verifypeer = 0) |> # SK seems to have verification error
             req_user_agent_getrad()
         }),
-        resp = httr2::req_perform_parallel(req,
-                                           paths = replicate(
-                                             length(req),
-                                             tempfile(fileext = ".h5", tmpdir = getwd())
-                                           )
+        # use rlang::.data to prevent note about no visible binding for global variable
+        resp = httr2::req_perform_parallel(rlang::.data$req,
+          paths = replicate(
+            length(rlang::.data$req),
+            tempfile(fileext = ".h5", tmpdir = getwd())
+          )
         ),
-        tempfile = purrr::map_chr(resp, purrr::chuck, "body"),
+        tempfile = purrr::map_chr(rlang::.data$resp, purrr::chuck, "body"),
```

Comment:
Note that purrr::map already has purrr::pluck shorthand built in: https://purrr.tidyverse.org/reference/map